linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_profile/oval/shared.xml

```diff
@@ -10,7 +10,8 @@
 
   <ind:textfilecontent54_object id="obj_umask_from_etc_profile"
   comment="Umask value from /etc/profile" version="1">
-    <ind:filepath>/etc/profile</ind:filepath>
+    <ind:filepath operation="pattern match">^\/etc\/profile(?:\.d\/.*\.sh|\.d\/sh\.local)?$</ind:filepath>
+    <!-- <ind:filepath>/etc/profile</ind:filepath> -->
```

Comment:
```suggestion
    <!-- <ind:filepath>/etc/profile</ind:filepath> -->
```
Is there any reason why this commeted out?

Reply:
Lack of memory. I forgot to remove. : )

Reply:
Fixed